### DISCUSSION TITLE
求助！！！ UX设计师遇到问题

### DISCUSSION THREAD

**Discussion by chenjianfang02** [Tencent/libpag#2039] [Q&A] [answered]
公司UX设计师用AE软件使用类似"loopOutDuration(type = "cycle", duration = 0)"函数导出PAG会报错。
有什么办法支持导出VectorComposition，同时支持函数

**Comment by kevingpqi123** [comment]
支持函数需要 pag sdk 层面开发，这里的开发周期会比较长
可以描述下你这边想实现的功能，看下是否可以通过其它方式折衷处理

**Reply by chenjianfang02** [reply]
方便加个微信吗？我拉上UX同事一起对下折中的方案
wx: 17603007934

如果不方便加的话，比如问题描述的循环播放一个动画，不用函数听UX同事说会要手动拖很多遍

**Comment by domchen** [comment, marked as answer]
目前并不支持表达式的直接导出，但是你可以右键点击一下图层，找到菜单将表达式转换为关键帧的功能点一下，就可以正常导出了。
